belgif-rest-problem-it/belgif-rest-problem-codegen-it/src/test/java/io/github/belgif/rest/problem/codegen/OpenApiGeneratorIT.java

```diff
@@ -0,0 +1,19 @@
+package io.github.belgif.rest.problem.codegen;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class OpenApiGeneratorIT {
```

Comment:
Should we maybe add a test that validates the config actually generated the 2 interfaces + 1 class for our definition? Exists or attempt to use them, both could work.

This would make sure that our configured mapping doesn't interfere with the generation of other classes.